store/index.js

```diff
@@ -245,6 +245,19 @@ export const getters = {
             p_state.dataDictionary.annotated[p_columnName].transformationHeuristic : "";
     },
 
+    getMappedCategories: (p_state) => (p_categorySkipList=[]) => {
+
+        // 1. Remove unmapped (null) columns and skipped categories
+        const currentCategories = Object.values(p_state.columnToCategoryMapping)
+            .filter(category => null !== category && !p_categorySkipList.includes(category));
+
+        // 2. Create a set of the unique mapped categories
+        const categorySet = new Set(currentCategories);
+
+        // Return the categories in array form
+        return [...categorySet];
+    },
+
     getMappedColumns: (p_state) => (p_category) => {
```

Comment:
> Because we have not yet had a discussion on how we want optional data dictionary key value pairs to be created, it is currently necessary to check for the existence of the missingValues list in both the getMappedColumns getter and changeMissingStatus mutation.

Referencing the PR description here, there is a check for `missingValues` in `changeMissingStatus` and `getUniqueValues` but not in `getMappedColumns`.

Reply:
The PR description mentions a check for `missingValues` in `getMappedColumns` but there isn't one instead there is a check in `getUniqueValues`.

Reply:
Ah! This is a typo. I'll amend it. It should read `getUniqueValues`. This refers to https://github.com/neurobagel/annotation_tool/pull/375/commits/83bd5114335d1e05a8ddf68b958a6d8dd5927ba6.

Reply:
Thanks for clarifying that!